challenge/index.md

```diff
@@ -163,6 +165,19 @@ We have invited senior leaders, scientists, researchers, and highly reputed mana
 * [Vito C. Ostuni](https://www.linkedin.com/in/vito-claudio-ostuni-0b576027){:target="_blank"}, Senior Research Scientist - Recommendation and Search - Netflix, Inc., Los Gatos, CA, USA
 * [Michele Ruta](http://sisinflab.poliba.it/ruta){:target="_blank"}, Professor of Information Systems - Polytechnic University of Bari, Italy
 * [Juan Sequeda](https://juansequeda.com){:target="_blank"}, Principal Scientist - data.world, Austin, TX, USA
+
+### Review criteria 
+
+Submitted apps will be evaluated according to the following criteria:
+
+* __creativity__: App idea is very unique or a novel take on other ideas, and/or a different mix of technologies/ components;
+* __design__: Excellent aesthetics, well beyond basic Punya typical UI. Inviting layout/colors, very clear how to use. Flows well;
+* __potential usefulness__: App demonstrates significant potential impact that relates to theme(s). Demonstrates it can effectively help the target audience;
+* __technical skill__: Well thought out and/or clever coding. Procedures used over duplicate code. Major app components are beyond common User Interface components used. Possible use of Any Component;
+* __presentation__: (video and supporting materials) Excellent use of video components to explain the app. Communicates well the importance and effectiveness of the app and its relation to theme(s);
```

Comment:
Correct the labels (e.g., creativity) to be capitalized consistent with the Linked Data items.

Reply:
Labels updated.